### DISCUSSION TITLE
Using a SFC with Formstep

### DISCUSSION THREAD

**Discussion by maxwellsmart84** [vueform/vueform#138] [Questions] [answered]
I was trying to create an SFC and then use that component inside a stepped form.  When I tried to give the SFC a `name` property so it could be filtered on, I noticed that `useVueform` will hijack this property and you cannot use the component inside the stepped form.  My fix for this was to pass it in as a prop, not use `useVueform` in my component at all, and pass the `name` prop to a GroupElement name inside my component.

I feel like I am doing something wrong here though.  I followed the guide here on Composition API:

https://vueform.com/docs/rendering-forms#using-composition-api

Not sure what I am doing wrong or if this is a known issue?

**Comment by adamberecz** [comment]
Could you create a minimal reproduction for this using eg. our [sandbox](https://codesandbox.io/p/sandbox/github/vueform/sandbox/tree/main)? I'd be happy to look into it then.

**Reply by maxwellsmart84** [reply, marked as answer]
I think I was able to work around this use ObjectElements.  I had a misunderstanding on how nested forms work - all set now.  Thanks.
